assets/sass/components/setup/_googlesitekit-setup-module.scss

```diff
@@ -126,9 +126,22 @@
 		> .googlesitekit-setup-module__input {
 			margin: 1em 0;
 		}
+	}
+
+	div[name="optimizeID"] {
+		margin-top: 20px;
+		width: 100%;
+
+		@media (min-width: $bp-wpAdminBarTablet) {
+			min-width: 350px;
+			width: auto;
+		}
```

Comment:
Why make it full-width in smaller viewports? It looks odd particularly because the textarea is not as wide. 
![image](https://user-images.githubusercontent.com/1621608/145368431-792749a4-6296-4600-8cb9-a5c22fd969ba.png)